config/canonical.json

```diff
@@ -23050,12 +23052,14 @@
       "shop": "convenience"
     }
   },
-  "shop/convenience|セーブオン": {
+    "shop/convenience|セーブオン": {
```

Comment:
This and all the other spaces look some kind of unintended... Could you maybe remove them?

Reply:
It should be enough to execute `npm run build`.

Reply:
Yep, seems like skipping the build. @higa4 please just run the build and push the changes 😁